packages/core/src/store.ts

```diff
@@ -1,18 +1,45 @@
-import { HoneybadgerStore } from './types';
+import { HoneybadgerStore, StoreContents } from './types';
+import { merge } from './util';
 
-export class GlobalStore<T> implements HoneybadgerStore<T> {
-  private store: T
+export class GlobalStore implements HoneybadgerStore {
+  private readonly contents: StoreContents;
+  private readonly breadcrumbsLimit: number;
 
-  constructor(store: T) {
-    this.store = store;
+  constructor(contents: StoreContents, breadcrumbsLimit: number) {
```

Comment:
Since we have a `static create` method, I think this constructor should be private. Otherwise, anyone can ignore can work around the `create` method and instantiate a `GlobalStore` with the constructor.

Reply:
Hmmm. Honestly, the `create` was added as more of a convenience than a restriction. I made a `create` method so we could "safely" instantiate the AsyncStore (return null if ALS not supported). Then I added it to the global store for consistency. We could make them private, but I don't care too much either way.

Reply:
OK, I guess this is fine since this class is really not used directly outside the package. If at some point we decide to do more in the `create` method before returning an instance, we could then consider making the constructor private.